File: Treinamento/Java/Estrutura_de_decisão/Questão01.java
import java.util.Scanner;
public class Questão01 {
    public static void main(String[] args) {
        Scanner s;
        s = new Scanner(System.in);

        System.out.print("Digite o primeiro número: ");
        int n1 = s.nextInt();

        System.out.print("Digite o segundo número: ");
        int n2 = s.nextInt();

        if (n1 > n2) {
            System.out.println(n1 + " é maior que " + n2);
        } else if (n1 < n2) {
            System.out.println(n2 + " é maior que " + n1);
        } else {
            System.out.println("Os números são iguais");
        }
    }
}

File: Treinamento/C/Estrutura_de_decisão/Questão01.c
#include <stdio.h>

int main() {
    int n1, n2;

    printf("Digite o primeiro número: ");
    scanf("%i", &n1);

    printf("Digite o segundo número: ");
    scanf("%i", &n2);

    if (n1 > n2) {
        printf("%i é maior que %i\n", n1, n2);
    } else if (n1 < n2) {
        printf("%i é maior que %i\n", n2, n1);
    } else {
        printf("Os números são iguais\n");
    }
}
